index.html

```diff
@@ -227,6 +231,262 @@ <h2 class="title-desktop">Multi-Post Stories</h2>
+              </button
+            ><button class="btn" type="button">See Source
+              <img src="./assest/git.png" alt="">
+            </button>
+          </div>
+          </div>
+        </div>
+
+        </div>      
+      </div>
+
+      <!--End card 3-->
+      <!--card 4-->
+      <div class="pop" data-target="p-4">
+        <i class="fas fa-times"></i>
+        
+        <div class="card-Content">
+          <h2 class="title">Multi-Post Stories</h2>
+          <h2 class="title-desktop">Multi-Post Stories</h2>
+          <div class="tech">
+            <span>CANOPY</span>
+            <span class="dot"></span>
+            <span>Back End Dev</span>
+            <span class="dot"></span>
+            <span>2015</span>
+          </div>
+          <img
+          class="card-image"
+          src="assest/pic4.PNG"
+          alt="Multi-Post Stories project"
+        />
+        <div class="desktop">
+          <p class="discription">
+            Lorem Ipsum is simply dummy text of the printing and typesetting
+            industry. Lorem Ipsum has been the industry's standard dummy text
+            ever since the 1500s, when an unknown printer took a galley of type
+            and scrambled it 1960s with the releaLorem Ipsum is simply dummy
+            text of the printing and typesetting industry. Lorem Ipsum has been
+            the industry's standard dummy text ever since the 1500s, when an
+            unknown printer took a galley of type and scrambled it 1960s with
+            the releorem Ipsum is simply dummy text of the printing and
+            typesetting industry. Lorem Ipsum han printer took a galley of type
+            and scrambled it 1960s with the releawn printer took a galley of
+            type and scrambled it 1960s with the releaLorem Ipsum is simply
+            dummy text of the printing and typesetting industry. Lorem Ipsum has
+            been the industry's standard dummy text ever since the 1500s, when
+            an unknown printer took a galley of type and scrambled it 1960s with
+            the relea
+          </p>
+          <div class="desktop-content">
+            <div class="lang">
+            <div class="feild">
+              <span>html</span>
+            </div>
+            <div class="feild">
+              <span>css</span>
+            </div>
+            <div class="feild">
+              <span>javaScript</span>
+            </div>
+          </div>
+          <div class="feild-button">
+            <button class="btn" type="button">See Live
+              <img src="./assest/Icon.png" alt="">
+              </button
+            ><button class="btn" type="button">See Source
+              <img src="./assest/git.png" alt="">
+            </button>
+          </div>
+          </div>
+        </div>
+        </div>
+      </div>
+      <!--End card 4-->
+    </div>
+    <!--The end of the popup cards-->
     <!--About myself section-->
```

Comment:
- [ ] You should implement a javascript that contains an array of objects to store JavaScript objects containing the information for each project.

- [ ]  Each object contains at least the following information about the project: name, description, featured image, technologies, link to live version, and link to the project source code.